package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "^0.16.0",
     "moment": "^2.18.1",
     "next": "^2.0.1",
+    "prop-types": "^15.5.7",
```

Comment:
react 15.5 deprecates using `React.PropTypes`, they recommend using the `prop-types` package.